src/transaction_builder.js

```diff
@@ -73,6 +73,13 @@ export class TransactionBuilder {
     }
     this.source = sourceAccount;
     this.operations = [];
+
+    if (isUndefined(opts.fee)) {
+      console.log(
+        '[TransactionBuilder] The `fee` parameter of `TransactionBuilder` is required. Future versions of this library will error if not provided.'
```

Comment:
Maybe add a package name here? Will be helpful to find what generated this message in larger projects.

Reply:
The challenge I was trying to work around is that most people would import `stellar-sdk`, and a few would import `stellar-base`, and we don't know at this juncture which they're importing. So I think it's better to be vague and let the dev make the connection, vs. naming the wrong one or naming both.